tests/contrib/gunicorn/wsgi_mw_app.py

```diff
@@ -1,20 +1,64 @@
+"""
+This app exists to replicate and report on failures and degraded behavior that can arise when using ddtrace with
+gunicorn
+"""
+import os
+
+
+if os.getenv("_DD_TEST_IMPORT_SITECUSTOMIZE"):
+    import ddtrace.bootstrap.sitecustomize  # noqa: F401  # isort: skip
+
+import json
+
 from ddtrace import tracer
 from ddtrace.contrib.wsgi import DDWSGIMiddleware
+from ddtrace.debugging import DynamicInstrumentation
+from ddtrace.internal.remoteconfig import RemoteConfig
+from ddtrace.profiling.collector.memalloc import MemoryCollector
+from ddtrace.profiling.profiler import Profiler
 from tests.webclient import PingFilter
 
 
+# starting the profiler here allows test cases to exercise the code paths that
+# restart threads preexisting a forked process
+Profiler._profiler = Profiler()
+Profiler._profiler.start()
```

Comment:
Why do we enable profiling in this way? this isn't normally how a user would enable.

```python
profiler = Profiler()
profiler.start()
```

?

or can do:

```python
import ddtrace.profiling.auto
```

It also looks like `Profiler._profiler` is meant to be a `_ProfilerInstance`, right?